ecllibrary/std/File.ecl

```diff
@@ -1040,4 +1040,29 @@ EXPORT varstring GetDefaultDropZone() :=
 EXPORT dataset(FsDropZoneRecord) GetDropZones() :=
     lib_fileservices.FileServices.GetDropZones();
 
+/**
+ * Return the expire days property of the specified logical filename.
+ *
+ * @param lfn           The name of the logical file.
+ 
+ 
+ * @return  	        Value of @expiredays properties of the file or -1 if that property not exists (never expire).
```

Comment:
trivial: remove 2 blank lines between @param and @return, and comments could line up.

Reply:
Removed